bagua/torch_api/distributed.py

```diff
@@ -386,10 +392,15 @@ def real_post_backward_hook(*unused):
             if not hasattr(optimizer, "_bagua_original_step"):
                 optimizer._bagua_original_step = optimizer.step
 
+            if not hasattr(optimizer, "_bagua_module"):
+                optimizer._bagua_module = self
```

Comment:
why do we need optimizer having the module attribute

Reply:
to skip optimizer step hook.